src/app/project/project-documents/detail/detail.component.ts

```diff
@@ -121,7 +141,14 @@ export class DocumentDetailComponent implements OnInit, OnDestroy {
       this.ngxSmartModalService.open('confirmation-modal');
   }
 
-  public openSnackBar(message: string, action: string) {
+  /**
+   * Opens a new snack bar notification message with a duration of 2 seconds, and executes an action
```

Comment:
```suggestion
   * Opens a new snack bar notification message with a duration of 2 seconds, and executes an action.
```
I'm guessing this was a cut and paste, because you were consistent at leaving out the period for all of them.. at least your consistent

Reply:
It's a virtue, I'm told